ddtrace/contrib/bottle/trace.py

```diff
@@ -58,7 +62,15 @@ def wrapped(*args, **kwargs):
                     code = 500
                     raise
                 finally:
-                    response_code = code or response.status_code
+                    if isinstance(result, HTTPResponse):
```

Comment:
is the ordering of these correct?

before you have `code or response.status_code`, but here you basically have the flip of that.

Reply:
The two orderings don't matter since either `code` or `result` is set, not both, since each is set within the `try:except`.